salt/version.py

```diff
@@ -96,107 +96,107 @@ class SaltStackVersion(object):
         'Nitrogen'      : (MAX_SIZE - 102, 0),
         'Oxygen'        : (MAX_SIZE - 101, 0),
         'Fluorine'      : (MAX_SIZE - 100, 0),
+        'Neon'          : (MAX_SIZE - 99, 0),
```

Comment:
All of what lines? Do you mean the ones that are commented out? Those all needed to be commented out because the linter does not like whitespace after arguments. I noticed this for Neon because it is the first one with a two-digit number in it, and decided to fix all of the other ones as well to avoid lint errors every time we have to uncomment a version.